inc/switchtec/mfg.h

```diff
@@ -102,6 +102,13 @@ struct switchtec_active_index {
 	enum switchtec_active_index_id keyman;
 };
 
+enum switchtec_bl2_recovery_mode {
+	SWITCHTEC_BL2_RECOVERY_I2C = 1,
+	SWITCHTEC_BL2_RECOVERY_XMODEM = 2,
+	SWITCHTEC_BL2_RECOVERY_I2C_AND_XMODEM = 3,
+	SWITCHTEC_BL2_RECOVERY_NOT_SET = 0xff
```

Comment:
I don't think SWITCHTEC_BL2_RECOVERY_NOT_SET makes any sense here... This is for the library interface and you seem to be abusing it for the CLI...

Reply:
Good idea. Removed definition and updated affected code.